roles/variables/defaults/main.yml

```diff
@@ -2,7 +2,7 @@
 # Custom filters used in this file are defined in plugins/filter/filters.py
 
 ### Version of Confluent Platform to install
-confluent_package_version: 6.2.1
+confluent_package_version: 7.0.0
```

Comment:
This should work given the cp-ansible tag for 7.0.0 is yet to be cut. Once the branch is cut for cp-ansible I believe this would be available anyway. @JumaX @domenicbove correct me here.

Reply:
i believe this breaks out build bc 7.0.0 is not public yet

Reply:
can y'all not run internal tooling against a development version of 7.0?  This is causing changes because yall check the version when determining which packages to uninstall, causing everything to be uninstalled every time something is installed, breaking colocation.
https://github.com/confluentinc/cp-ansible/blob/ba68ede187b33b0b60ab37f6f9c6ce9765023dcd/roles/common/tasks/remove_packages.yml#L7

Reply:
the major problem is that packages.confluent.io does not have the 7.0.0 packages currently. so updating that variable here would lead to issues.

If you have a automated tool that installs 7.0.0, you must be using variables to achieve this. and you can just add `confluent_package_version: 7.0.0` to those variables